client/components/fixed-navigation-header/index.tsx

```diff
@@ -0,0 +1,78 @@
+import styled from '@emotion/styled';
+import classNames from 'classnames';
+import { ReactNode } from 'react';
+import { preventWidows } from 'calypso/lib/formatting';
+
+const Header = styled.header`
+	position: fixed;
+	z-index: 1;
+	top: var( --masterbar-height );
+	left: var( --sidebar-width-max );
+	width: calc( 100% - var( --sidebar-width-max ) );
+	padding: 0 32px;
+	box-sizing: border-box;
+	border-bottom: 1px solid var( --studio-gray-5 );
+	background-color: #fff;
```

Comment:
We do have a variable called ` --studio-white` that could be used here:

```css
background-color: var( --studio-white )
```

Reply:
Great catch! Fixed in 5f46a81